portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,103 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+}
+
+button {
+  background: none;
+	border: none;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+a:hover {
+  color: #ede7f6;
+}
+
+.nav-item {
+  color: #b39ddb;
+  font-size: 1.25rem;
+  font-family: 'Inconsolata', monospace;
+  margin: 0px 10px;
+}
+
+.selected {
+  color: #ede7f6;
+}
+
+.sticky {
```

Comment:
sticky isn't the best name for this because there's a [position: sticky]

Reply:
Changed to stick-to-top. Thanks!